tests/connmap.py

```diff
@@ -14,12 +14,24 @@
 RE_MAC = rb"(?P<mac>([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2})"
 
 
+class MapClient:
+    '''Represents a client (STA) in the connection map.'''
+    def __init__(self, mac: str):
+        self.mac = mac
+
+
```

Comment:
Extra line break - doesn't flake8 complain about this?

Reply:
I don't see the extra line break... PEP8 specifies two empty lines before each top-level class or function definition.